earth_enterprise/src/server/wsgi/wms/ogc/common/geom.py

```diff
@@ -44,6 +44,12 @@ def __div__(self, b):
       return Pair(self.x / b.x, self.y / b.y)
     else:
       return Pair(self.x / b, self.y / b)
+  
+  def __truediv__(self, b):
+    if isinstance(b, Pair):
+      return Pair(self.x // b.x, self.y // b.y)
```

Comment:
Any reason why this uses // instead of the / that the other functions use?

Reply:
I think this could use some comments

Reply:
It needs fixed. Its supposed to just be standard division. I saw that earlier today thinking I forgot to fix it, but it was actually correct  in the first place.